### ISSUE TITLE
disable uncheckAll  & limitSelection min 

### ISSUE THREAD

**Issue by Amor-Fairouz** [NileshPatel17/ng-multiselect-dropdown#83]
hi! 
i'm using this module and i'm searching how to disable the option 'DeselectAll' or to get an option limitSelection=1 
any help ?

**Comment by NileshPatel17** [comment]
you can checkout limitSelction example here
https://nileshpatel17.github.io/ng-multiselect-dropdown/

**Comment by Amor-Fairouz** [comment]
i need the selectAll option and i need to hide just the deselectAll any idea how to do it ?

**Comment by NileshPatel17** [comment]
thats a strange use case. You want select all but not deselectall. Its not possible.

**Comment by Amor-Fairouz** [comment]
yeah maybe ;D anyway I set the OnItemDeselect method to limit the selection of list to one item 
 OnItemDeSelect(item: any) {
    if (this.selectedItems.length === 0) {
      this.selectedItems = [item];
    } else {
    this.userSelectedItems.emit(this.selectedItems);
    }
  }

**Comment by NileshPatel17** [comment]
Please share sample code with https://stackblitz.com or codesandbox. Let me see if i can help you.

**Comment by NileshPatel17** [comment]
Closing this.

**Comment by omkarg18** [comment]
Hi any updates on this,
Here is the case I want one default value selected and want prevent to deselect all.
Atleast one item should be selected.
I tried using onDeselectAll($event){ event.preventdefault()
 }

But no event was captured amd prevented.
Any suggestions?

**Comment by NileshPatel17** [comment]
@omkarg18 I can help you only if you share sample code using any online editor such as codesandbox.
